### PR TITLE
add beer-song

### PR DESCRIPTION
**Pull request by devonestes**
I'm just dibsing this exercise for now and I'll add the implementation and test suite soon.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -56,7 +56,8 @@
     "dot-dsl",
     "custom-set",
     "nth-prime",
-    "forth"
+    "forth",
+    "beer-song"
```
Comment (dalexj) [inline config.json]: I dont think this exercise belongs at the end of this list

File: exercises/beer-song/beer_song.exs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+defmodule BeerSong do
+  def verse(number) do
+    # Your implementation here...
+  end
+
+  def lyrics(range) do
+    # Your implementation here...
+  end
```
Comment (dalexj) [inline exercises/beer-song/beer_song.exs]: can you add some typespecs and docs?

File: exercises/beer-song/example.exs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+defmodule BeerSong do
+  def verse(number) when number == 1 do
```
Comment (dalexj) [inline exercises/beer-song/example.exs]: here's a simpler way to do this:

```elixir
def verse(1) do
  ..
end
```

File: config.json (hunk 1 above)
Comment (devonestes) [inline config.json]: Does that list determine the order in which people get the exercises when the call `exercism fetch elixir`? If so, it totally doesn't!

File: exercises/beer-song/beer_song.exs (hunk 2 above)
Comment (devonestes) [inline exercises/beer-song/beer_song.exs]: Sure thing! Should they be doc tests, or just plan ol' docs?

File: exercises/beer-song/example.exs (hunk 3 above)
Comment (devonestes) [inline exercises/beer-song/example.exs]: Brilliant - thanks for this!

File: config.json (hunk 1 above)
Comment (kytrinyx) [inline config.json]: Yeah, the `config.json` determines the order in which people receive exercises by default. (We really need to make that easier to understand/discover!)

**Comment by parkerl** [comment]
Thanks! 🎉

**Pull request merged**